BankexWallet/InterfaceElements/DesignElements/WalletCreationTypeController/WalletCreationTypeController.swift

```diff
@@ -11,12 +11,16 @@ import UIKit
 class WalletCreationTypeController: UIViewController {
 
     @IBOutlet weak var backButton: UIButton!
+    var isBackButtonHidden: Bool?
```

Comment:
You can omit optionality and make default false

Reply:
fixed